cli/tests/integrations/test_service.py

```diff
@@ -197,16 +201,28 @@ def test_log_config():
 
 def test_log_follow():
     wait_for_service('chronos')
-    proc = subprocess.Popen(['dcos', 'service', 'log', 'chronos', '--follow'],
-                            preexec_fn=os.setsid,
-                            stdout=subprocess.PIPE,
-                            stderr=subprocess.PIPE)
+    args = ['dcos', 'service', 'log', 'chronos', '--follow']
```

Comment:
Can we write a comment as to why we are doing this? I know that we talked about this extensively and I already forgot! 💭